test/server.go

```diff
@@ -21,8 +19,6 @@ func CoreDNSServer(corefile string) (*caddy.Instance, error) {
 	defer mu.Unlock()
 	caddy.Quiet = true
 	dnsserver.Quiet = true
-	log.SetOutput(ioutil.Discard)
```

Comment:
This line was useless before this commit ?
I mean : what impact for existing Test that are using this function ?

Reply:
This is redundant and has no impact on existing tests.